### ISSUE TITLE
lang attribute in metadata and nav doc

### ISSUE THREAD

**Issue by gregoriopellegrino** [daisy/epub-accessibility-tests#64]
Proposal to create a test to ensure that the lang and xml:lang attributes present in metadata (`content.opf`) and in the navigation file (TOC) digested by the reading application are respected and correctly passed to the AT.

Ensuring that the lang and xml:lang attributes are present and correctly passed on to users is essential for accessibility. These attributes specify the language of the content, which is crucial for users who are visually impaired or have reading disabilities. Screen readers use the lang and xml:lang attributes to determine the language to use when reading the content to the user. If these attributes are missing or incorrect, it can result in a poor user experience for the user, making the content difficult or impossible to understand.

**Comment by avneeshsingh** [comment]
Screen readers use lang attribute of HTML file.
I do not think that screen readers are effected by lang attribute of OPF file. It is mainly used by reading systems.

**Comment by gregoriopellegrino** [comment]
I agree, to better explain the issue I present two cases we've seen in some reading applications:

- title of the publication is in Italian (with xml:lang attribute in the OPF file set as Italian) displayed in an English User Interface (in the library view), it is read out with English voice by the screen-reader: it seems that the reading system doesn't pass the language information of the title string to the assistive technologie
- the book is in English, but a chapter in the book as a French heading (is it correctly marked as French in the nav doc), in the TOC User Interface it is read out with English voice by the screen-reader: it seems that the reading system doesn't pass the language information of the nav element string to the assistive technology

**Comment by avneeshsingh** [comment]
OK, this issue is mainly about reading system not exposing lang attributes in its widgets.

**Comment by GeorgeKerscher** [comment]
Adding this to the 2023 milestones.

**Comment by ways2read** [comment]
Moved to 2024 milestone in meeting 20/02/2024

**Comment by marisademeglio** [comment]
Noting that although we have two language switching tests now:
* [non-visual](https://github.com/daisy/epub-accessibility-tests/blob/main/content/epub30-test-0302/EPUB/xhtml/Non_Visual_Reading_Tests.xhtml#L393)
* [advanced read aloud](https://github.com/daisy/epub-accessibility-tests/blob/main/content/epub30-test-0370/EPUB/xhtml/advanced_read_aloud_tests.xhtml#L13)

@gregoriopellegrino 's [points above](https://github.com/daisy/epub-accessibility-tests/issues/64#issuecomment-1609390804) are not covered by them.
